src/components/CertificatesList/messages.js

```diff
@@ -26,13 +26,20 @@ const messages = defineMessages({
     defaultMessage: 'Verifiable Credentials',
     description: 'Header for the Verifiable Credentials page',
   },
-  credentialsDescription: {
+  programCredentialsDescription: {
     id: 'credentials.description',
     defaultMessage:
       'A certificate for a program will appear in the list once you have earned all course certificates in a program.',
     description:
       'Description of program credentials for the Verifiable Credentials page',
   },
+  courseCredentialsDescription: {
+    id: 'credentials.description',
+    defaultMessage:
+      'A certificate for a course will appear once you have completed course.',
```

Comment:
```suggestion
      'A course certificate will appear once you have completed a course.',
```

This is only a suggestion, but I was trying to rewrite this so that the end of the sentence made it clear that a certificate would be received once the learner finished _any_ course. 
Another way to write this is: 
"A certificate for a course will appear one you have completed the course."

Reply:
Changed text to `A course certificate will appear once you have completed a course.`
https://github.com/openedx/frontend-app-learner-record/compare/140ec21935491fbe9a1523a62f3314cc3480bac7..4f0a0eaa7f56173c8e88064f482bb1fece045256